frontend/src/routes.js

```diff
@@ -53,10 +52,6 @@ export const ROUTES = {
       path: "/evaluation/self",
       component: SelfEvaluation
     },
-    otherevaluation: {
```

Comment:
Don't we need to add evaluation for family members ?

Reply:
Think I misunderstood something. I'll recheck the designs

Reply:
@surdu  done, you can take a look again